test/bigi.js

```diff
@@ -6,72 +7,76 @@
 var assert = require('assert')
 var BigInteger = require('../')
 
-describe('BigInteger', function () {
-  it('should work without new', function() {
-    var bi = BigInteger('12345')
-    assert.equal(bi.toString(10), '12345')
-  })
+function newBI (str, radix) {
+  if (str[0] === '-') {
+    if (radix === 16) return BigInteger.fromString('' + parseInt(str, 16))
+  }
```

Comment:
We don't test any numbers < -MAX_SAFE_INT,  and `fromHex` only supports unsigned hex